src/Core/Termination.idr

```diff
@@ -186,6 +189,14 @@ mutual
                  do scs <- traverse (findSC defs env Unguarded pats) args
                     pure (concat scs)
       where
+        handleCase : Term vars -> List (Term vars) -> Core (Maybe (List SCCall))
+        handleCase (Ref fc nt n) args
+            = do n' <- toFullNames n
+                 if caseFn n'
```

Comment:
```suggestion
            = if caseFn !(toFullNames n)
```
What about this?

Reply:
Possibly, but it elaborates to the same, and any day now there'll be another check on n' to see if it's the "if..then..else" function.